File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;
import entity.Result;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    public PageResult findPage(int pageNum,int rows);

    public void add(TbBrand tbBrand);

    public TbBrand findOne(Long id);

    public void update(TbBrand tbBrand);

    public void delete(long[] ids);

    public PageResult serch(TbBrand tbBrand,int pageNum,int pageSize);

    public List<Map> findBrandList();
}

File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    this.findAll=function () {
       return $http.get("../brand/findAll.do");
    }
    this.findPage=function(page, rows) {
        return $http.get("../brand/findPage.do?page="+page+"&rows="+rows);
    }
    this.save=function(entity) {
        return $http.post("../brand/add.do",entity);
    }
    this.findOne=function(id) {
        return $http.get("../brand/findOne.do?id="+id);
    }
    this.del=function(ids) {
        return $http.get("../brand/del.do?ids="+ids);
    }
    this.search=function(page, rows,searchEntity) {
        return $http.post("../brand/search.do?page="+page+"&rows="+rows,searchEntity);
    }
    this.findBrandList=function () {
        return $http.get("../brand/findBrandList.do");
    }
})